components/jackal_remove_miniitx.mdx

```diff
@@ -0,0 +1,44 @@
+1. If you have a mini-ITX computer installed it will need to be removed. With the computer tray
+   open locate each of the cables connected to the motherboard and hard drive. Remove the power,
+   power switch, USB and SATA cables. The two cable ties can be cut to completely remove the power
+   and SATA cables. The two antenna cables are connected underneath. Keep this in mind for the
+   next steps.
+
+    <center>
+      <figure>
+        <img
+          src="/img/computer_images/jackal_pc_removal/1.jpg"
+          width="600"
+        />
+        <figcaption>Disconnecting cables</figcaption>
+      </figure>
+    </center>
+
+2. Remove the four (4) hex screws from the motherboard using a 2.5mm wrench. Gently lift it out
```

Comment:
Change to 'Remove the 4 hex screws...'
Change to '2.5 mm'

Reply:
Updated